osgi.specs/docbook/core/058/service.resolver.xml

```diff
@@ -1415,6 +1416,144 @@ public List&lt;Capability&gt; findProviders( Requirement requirement) {
     </section>
   </section>
 
+  <section>
+    <title>Obtaining Resolvers Using SPI</title>
+
+    <para>While the <xref linkend="org.osgi.service.resolver.Resolver"
+    xrefstyle="hyperlink"/> service is typically obtained from the OSGi
+    service registry when running within an OSGi framework, there are many
+    scenarios where a resolver is needed outside an OSGi runtime:</para>
+
+    <itemizedlist>
+      <listitem>
+        <para><emphasis>Build Tools</emphasis> - Build tools such as Maven,
+        Gradle, or bnd need to resolve project dependencies at build time,
+        often modeling non-OSGi artifacts (like Maven dependencies) as
+        resources with requirements and capabilities.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis>IDEs and Development Tools</emphasis> - Integrated
+        development environments need to analyze bundle dependencies and
+        predict resolution outcomes without running a full OSGi
+        framework.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis>Provisioning Tools</emphasis> - Tools that operate on
```

Comment:
I don't think we have used the term `Provisioning Tool` anywhere else in the specification.  I would use `Management Agent` instead:
```suggestion
        <para><emphasis>Management Agents</emphasis> - Agents that operate on
```